src/js/components/PageHeader.js

```diff
@@ -90,15 +90,25 @@ class PageHeader extends React.Component {
               </div>
             </div>
           </div>
-          {children}
+          {navigationTabs}
         </div>
       </div>
     );
   }
 }
 
+PageHeader.defaultProps = {
+  className: 'container container-fluid container-pod container-pod-short flush flush-top',
```

Comment:
This is probably an artifact of rebasing but we don't need these default classnames anymore since we are utilising the subtractive feature of `classnames` now